packages/contracts/src/governance/base/Addresslist.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity ^0.8.8;
+
+import {CheckpointsUpgradeable} from "@openzeppelin/contracts-upgradeable/utils/CheckpointsUpgradeable.sol";
+import {_uncheckedAdd, _uncheckedSub} from "@aragon/osx/utils/UncheckedMath.sol";
+
+/// @title Addresslist
+/// @author Aragon Association - 2021-2024
```

Comment:
```suggestion
/// @author Aragon X - 2021-2024
```
Since this is a forked version with changes, we should mention here Aragon X. We can potentially change them all in this repo, but definitely not done by Aragon Association.

Reply:
Done